debezium-core/src/main/java/io/debezium/pipeline/AbstractChangeRecordEmitter.java

```diff
@@ -18,10 +19,12 @@
  */
 public abstract class AbstractChangeRecordEmitter<T extends DataCollectionSchema> implements ChangeRecordEmitter {
 
+    protected final Partition partition;
```

Comment:
Let's make this private and access via the public getter. I know we do use field access in other places, and it's a bit inconsistently handled across the board. But let's try for consistency within a single class at least.

Reply:
Done.